### PR TITLE
Actualize Wazuh Puppet module

### PR DESCRIPTION
**Pull request by Hexta**
Update references to the class wazuh::manager.
Correct name of parameter wazuh::manager:: ossec_smtp_server.

### CODE REVIEW

File: source/deploying-with-puppet/wazuh-puppet-module/index.rst [hunk 1]
```diff
@@ -144,5 +144,5 @@ Reference Wazuh puppet
     :maxdepth: 1
 
     reference-wazuh-puppet/ossec-scanpaths
-    reference-wazuh-puppet/wazuh-agent-class
+    reference-wazuh-puppet/wazuh-manager-class
```
Comment (jm404) [inline source/deploying-with-puppet/wazuh-puppet-module/index.rst]: The `wazuh-manager-class` should override `wazuh-server-class` and `wazuh-agent-class` should be left intact.

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst [hunk 2]
```diff
@@ -1,14 +1,14 @@
 .. Copyright (C) 2019 Wazuh, Inc.
 
-.. _reference_wazuh_server_class:
+.. _reference_wazuh_manager_class:
```
Comment (jm404) [inline source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst]: @alberpilot please confirm if changing file name requires to update some references or not.

Comment (alberpilot) [reply source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst]: Not necessary. Thanks, both

File: source/deploying-with-puppet/wazuh-puppet-module/index.rst (hunk 1 above)
Comment (Hexta) [reply source/deploying-with-puppet/wazuh-puppet-module/index.rst]: It is my fault.
Fixed.

Comment (jm404) [reply source/deploying-with-puppet/wazuh-puppet-module/index.rst]: Great thanks

**Comment by alberpilot** [comment]
Thanks @Hexta for your contribution. 
GJ @jm404 for reviewing!

**Pull request merged**